enable __FILE__ trim (#1401)
oneapi-src/oneTBB: cmake/README.md

TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH - Disable HWLOC automatic search by pkg-config tool (OFF by default)
TBB_ENABLE_IPO - Enable Interprocedural Optimization (IPO) during the compilation (ON by default)
TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, only available on the Apple platform. (OFF by default)

> TBB_FILE_TRIM - Enable __FILE__ trim (ON by default)

Reviewer: To be honest, "Enable FILE trim" still doesn't say much about what this option does. Can you add more description about usage of relative paths instead of full path.

Author: More description has been added.

Reviewer: @aepanchi Could you please take a look?

TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH - Disable HWLOC automatic search by pkg-config tool (OFF by default)
TBB_ENABLE_IPO - Enable Interprocedural Optimization (IPO) during the compilation (ON by default)
TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, only available on the Apple platform. (OFF by default)

> TBB_FILE_TRIM - Enable __FILE__ trim, replace a build-time full path with a relative path in the debug info and macro __FILE__, this can be used to make

Reviewer 2: ```suggestion
TBB_FILE_TRIM - Enable __FILE__ trim, replace a build-time full path with a relative path in the debug info and macro __FILE__; use it to make
```

>            reproducible builds that are location independent (ON by default)

Reviewer 2: ```suggestion
           reproducible location-independent builds (ON by default)
```
